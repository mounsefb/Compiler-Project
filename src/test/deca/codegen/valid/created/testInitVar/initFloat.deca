// Description :
//    Test de la déclaration d'un float sans initialisation
//
// Résultats :
//    OK


{
    float y = 7;
    println(y);
}


