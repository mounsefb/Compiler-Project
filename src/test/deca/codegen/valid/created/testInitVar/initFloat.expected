7.00000e+00
